src/emoji/EmojiPickerScreen.js

```diff
@@ -28,13 +40,27 @@ type Props = $ReadOnly<{|
 
 type State = {|
   filter: string,
+  numColumn: number,
 |};
 
 class EmojiPickerScreen extends PureComponent<Props, State> {
   state = {
     filter: '',
+    numColumn: Dimensions.get('window').width / 44 - 2,
```

Comment:
[Magic number](https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad#:~:text=In%20programming%2C%20a%20magic%20number,is%20changed%20and%20not%20another.) `44 -2`? can you explain the calculation of this? Maybe assign it to a const with a proper name?